Sources/Sentry/SentryTracer.m

```diff
@@ -92,6 +98,12 @@ - (instancetype)initWithTransactionContext:(SentryTransactionContext *)transacti
             initSlowFrames = currentFrames.slow;
             initFrozenFrames = currentFrames.frozen;
         }
+#endif
+#if SENTRY_TARGET_PROFILING_SUPPORTED
+        if ([_hub getClient].options.enableProfiling) {
+            _profiler = [[SentryProfiler alloc] init];
+            [_profiler start];
```

Comment:
`h`: This will allocate and start a profiler for every transaction. Can the profilers run in parallel? Is this what we want?

Reply:
will fix, thanks for the heads up